docs/reference.md

```diff
@@ -106,9 +106,11 @@ be fed to the program, and generates output data values, which are matched
 against the user-provided expected data values. The data values (B) are
 hard-coded in the program itself using `stablehlo.constant` operations. The
 interpreter evaluates the input program. The output(s) of the op under test
-is checked via checks (e.g. `check.eq`, `check.almost_eq`), as shown below.
-`check.eq` checks for bitwise equality for any supported type and
-`check.almost_eq` checks for nearly equality within a tolerance,
+is checked via checks (e.g. `check.expect_eq`, `check.expect_almost_eq`), as
+shown below.
+`check.expect_eq` and `check.expect_eq_const` checks for bitwise equality for
```

Comment:
Typo: "check" (not "checks", because there are multiple nouns now). Here and below.